### DISCUSSION TITLE
Pytanie o migracje legacy apps do kontenerów

### DISCUSSION THREAD

**Discussion by rangerek** [AzureCommunityPL/group#75] [Pomoc]
Witam
Czy jest tu ktoś, kto w praktyce przeszedł przez migracje aplikacji legacy (win 2008 albo starsze) do kontenerów - najlepiej Docker Enterprise 
I jest chętny podzielić się kilkoma szczegółami i odpowiedzieć na pare high level pytań :)

**Comment by smereczynski** [comment]
Jak ma się to do Azure? Jeśli nijak, to usuń proszę post.
